aries-site/src/examples/components/layer/LayerStickyFormExample.js

```diff
@@ -0,0 +1,164 @@
+              value={formValues}
+              onChange={onFormChange}
+              method="post"
+            >
+              <Box margin={{ bottom: 'small' }}>
+                <FormField name="monitorName" label="Monitor Name">
+                  <TextInput name="monitorName" />
+                </FormField>
+                <FormField name="desc" label="Bussiness Description">
+                  <TextArea name="monitorName" />
+                </FormField>
+                <Text
+                  margin={{ vertical: 'medium' }}
+                  weight="bold"
+                  color="text-strong"
+                >
+                  Monitor Detail
+                </Text>
+                <FormField name="monitorType" label="Monitor Type">
+                  <TextInput name="monitorType" />
+                </FormField>
+                <FormField name="rules">
+                  <CheckBox
+                    name="rules"
+                    label="Include Process Rules"
+                    toggle
+                    fill
+                    checked
+                    reverse
+                  />
+                </FormField>
+                <FormField name="frameworks" label="Frameworks">
+                  <Select options={options} name="frameworks" />
+                </FormField>
+                <FormField name="severity" label="Rule Severity">
+                  <Select options={options} name="severity" />
+                </FormField>
+                <FormField name="provider" label="Provider">
+                  <Select options={options} name="provider" />
+                </FormField>
+                <FormField name="account" label="Account">
+                  <Select options={options} name="account" />
+                </FormField>
+                <FormField name="ruleName" label="Rule Name">
+                  <Select options={options} name="ruleName" />
+                </FormField>
+                <Text
+                  margin={{ vertical: 'medium' }}
+                  weight="bold"
+                  color="text-strong"
+                >
+                  Email Notification
+                </Text>
+                <FormField name="notifications">
+                  <CheckBox
+                    name="notifications"
+                    label="Email Notifications"
+                    toggle
+                    fill
+                    checked
+                    reverse
+                  />
+                </FormField>
+                <FormField name="email" label="Primary Email">
+                  <TextInput name="email" />
+                </FormField>
+              </Box>
+              <Box>
+                <Box
+                  direction="row"
+                  gap="small"
+                  align={
+                    !['xsmall', 'small'].includes(size) ? 'start' : undefined
+                  }
+                  margin={{ top: 'small', bottom: 'small' }}
```

Comment:
```suggestion
                  pad={{ vertical: 'small' }}
```
We have been trying to move away from using margin and using pad instead.